Modules/System/SharePoint/src/SharePointClient.Codeunit.al

```diff
@@ -266,6 +266,15 @@ codeunit 9100 "SharePoint Client"
     procedure DownloadFileContent(OdataId: Text; FileName: Text): Boolean
     begin
         exit(SharePointClientImpl.DownloadFileContent(OdataId, FileName));
```

Comment:
end;

Reply:
My aplologies, I tried to multitask while doing this. Generally a bad idea. ;-)

An update is coming.